tasks/main.yml

```
@@ -9,28 +17,58 @@
     state: "{{ foreman_scap_client_repo_state }}"
   when: ansible_os_family == "RedHat"
 
-- name: Configure plugins repository (apt)
-  apt_repository:
-    repo: "{{ foreman_scap_client_apt_repo_url }}"
-    state: "{{ foreman_scap_client_repo_state }}"
-  when: ansible_os_family == "Debian"
+- name: Configure client repository (apt) - sources.list format
```

```suggestion
- name: Configure client repository (zypper)
  block:
    - name: Configure plugins repository (zypper)
      community.general.zypper_repository:
        name: "foreman-plugins-{{ foreman_scap_client_release }}"
        description: "Foreman plugins repository"
        repo: "{{ foreman_scap_client_repo_url }}"
        state: "{{ foreman_scap_client_repo_state }}"
        disable_gpg_check: '{{ foreman_scap_client_repo_gpg | ternary("false", "true") }}'
  when: ansible_facts.os_family == "Suse"

- name: Configure client repository (apt) - sources.list format
```

Adding repo for zypper